tests/integration_tests/without_db/udfs/test_token_classification_udf.py

```diff
@@ -0,0 +1,98 @@
+import pandas as pd
+from typing import Dict
+import pytest
+import torch
+
+from exasol_transformers_extension.udfs.models.token_classification_udf import \
+    TokenClassificationUDF
+from tests.utils.parameters import model_params
+from exasol_udf_mock_python.connection import Connection
+
+
+class ExaEnvironment:
+    def __init__(self, connections: Dict[str, Connection] = None):
+        self._connections = connections
+        if self._connections is None:
+            self._connections = {}
+
+    def get_connection(self, name: str) -> Connection:
+        return self._connections[name]
+
+
+class Context:
+    def __init__(self, input_df):
+        self.input_df = input_df
+        self._emitted = []
+        self._is_accessed_once = False
+
+    def emit(self, *args):
+        self._emitted.append(args)
+
+    def reset(self):
+        self._is_accessed_once = False
+
+    def get_emitted(self):
+        return self._emitted
+
+    def get_dataframe(self, num_rows='all', start_col=0):
+        return_df = None if self._is_accessed_once \
+            else self.input_df[self.input_df.columns[start_col:]]
+        self._is_accessed_once = True
+        return return_df
+
+
+@pytest.mark.parametrize("description,  device_id, n_rows, agg_strategy", [
+    ("on CPU with batch input with none aggregation", None, 3, "none"),
+    ("on CPU with batch input with NULL aggregation", None, 3, None),
+    ("on CPU with batch input with max aggregation", None, 3, "max"),
+    ("on CPU with single input with none aggregation", None, 1, "none"),
+    ("on CPU with single input with NULL aggregation", None, 1, None),
+    ("on CPU with single input with max aggregation", None, 1, "max"),
+    ("on GPU with batch input with none aggregation", 0, 3, "none"),
+    ("on GPU with batch input with NULL aggregation", 0, 3, None),
+    ("on GPU with batch input with max aggregation", 0, 3, "max"),
+    ("on GPU with single input with none aggregation", 0, 1, "none"),
+    ("on GPU with single input with NULL aggregation", 0, 1, None),
+    ("on GPU with single input with max aggregation", 0, 1, "max")
+])
+def test_token_classification_udf(
```

Comment:
I think, another test with multiple aggregation_strategies would be helpful here, to check the assumption that we can specify the aggregation_strategy during call

Reply:
We are already able to introduce this parameter in the call; we would get an error if we couldn't.

Reply:
This test is for testing, if we can change the strategy, sometimes, you can set it ones, but not change it later.